archivist.py

```diff
@@ -40,9 +47,18 @@
         path = challenge_path
 
 for challenge_path in challenge_paths:
-    print(challenge_path)
+    if args.mode == "skip" and os.path.exists(challenge_path):
+            print(f"{LIGHTBLUE}[skip]{RESET} " + challenge_path)
+            continue
+    elif args.mode == "duplicate" and os.path.exists(challenge_path):
+            if os.path.exists(challenge_path + "_copy"):
+                rmtree(challenge_path + "_copy")
+            os.rename(challenge_path, challenge_path + "_copy")
```

Comment:
I think `..._copy` should be the new copy instead of being the original copy

Reply:
`copy` might be confusing. I think `backup` is what i want . The idea is if someone has solved/unsolved challenge and wants to get a new copy of the challenge (for any reason), but don't want to lose the progress. The backup mode will do the job!